### PR TITLE
#82 - add community standards files

### LINKED ISSUE #82: Add repository documentation
I would like to see:
- [ ] licence file
- [ ] polished readme file
- [ ] contribution guidelines file
- [ ] changelog file

### PR DESCRIPTION
**Pull request by JakubKermes**
This should close #82.

### CODE REVIEW

**Comment by JakubKermes** [comment]
@krzysztofrewak Should we host website with documentation for BLT?

**Comment by krzysztofrewak** [comment]
> @krzysztofrewak Should we host website with documentation for BLT?

@JakubKermes, it's hosted in Github Pages: https://blumilksoftware.github.io/blt/

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]
Please lowercase filenames `changelog.md`, `contributing.md` and rename licencefile to `licence.md`

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+2. [Style Guides](#style-guides)
+   - [Git Commit Messages](#git-commit-messages)
+   - [Code Style](#code-style)
+3. [Getting Help](#getting-help)
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+If you find a bug, please open an issue [here](https://github.com/yourusername/yourproject/issues) and include as much detail as possible. Before creating a new issue, please check to see if the bug has already been reported.
```
Comment (EwelinaSkrzypacz) [inline contributing.md:20]:
```suggestion
If you find a bug, please open an issue [here](https://github.com/blumilksoftware/blt/issues) and include as much detail as possible. Before creating a new issue, please check to see if the bug has already been reported.
```

File: docs/elements/cookies.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+     * @Then the following cookies should be present:
+     * @Then the cookies should be set to:
+     */
+    public function assertCookiesExist(TableNode \$table): void
+    {
+        foreach (\$table as \$row) {
+            \$cookieValue = \$this->request->cookies->get(\$row['name']);
+            Assert::assertEquals(\$row['value'], \$cookieValue, 'Cookie {\$row['name']} does not have the expected value {\$row['value']}.');
+        }
+    }"></code-snippet>
```
Comment (EwelinaSkrzypacz) [inline docs/elements/cookies.html:106]:
```suggestion
    }"></code-snippet>
    
```

File: docs/elements/hooks.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+        public function __construct()
+        {
+        $bootstrapper = new LaravelBootstrapper();
+        $bootstrapper->boot();
+        }
+        }
+    </code-block>
+
+</article-paragraphs>
```
Comment (EwelinaSkrzypacz) [inline docs/elements/hooks.html:56]:
```suggestion
</article-paragraphs>

```

**Comment by EwelinaSkrzypacz** [comment]
Conflicts :eyes:

File: docs/elements/hooks.html [hunk 4]
```diff
@@ -53,4 +53,5 @@
         }
     </code-block>
 
-</article-paragraphs>
\ No newline at end of file
+</article-paragraphs>
+
```
Comment (EwelinaSkrzypacz) [inline docs/elements/hooks.html:57]: Now it's to much empty lines on the end of the file :eyes:

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Review by krzysztofrewak — APPROVED** [review]

**Pull request merged**